<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Source</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>URL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="urlEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Filename parameter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filenameParameterEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>First</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="firstSpinBox">
           <property name="maximum">
            <number>2000000000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Last</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="lastSpinBox">
           <property name="maximum">
            <number>2000000000</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Step</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="stepSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="detectFromURLButton">
           <property name="text">
            <string>Detect from URL</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Referrer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="refererEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>User-agent</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="userAgentEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Destination</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="folderEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="useLastDirectoryCheckBox">
         <property name="text">
          <string>Use last directory from URL as local directory</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="replaceUnderscoresBySpacesCheckBox">
         <property name="text">
          <string>Replace underscores by spaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="skipCheckBox">
         <property name="text">
          <string>Skip existing files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="stopCheckBox">
         <property name="text">
          <string>Stop on error</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="downloadButton">
         <property name="text">
          <string>Download</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>urlEdit</tabstop>
  <tabstop>filenameParameterEdit</tabstop>
  <tabstop>firstSpinBox</tabstop>
  <tabstop>lastSpinBox</tabstop>
  <tabstop>stepSpinBox</tabstop>
  <tabstop>detectFromURLButton</tabstop>
  <tabstop>refererEdit</tabstop>
  <tabstop>userAgentEdit</tabstop>
  <tabstop>folderEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>useLastDirectoryCheckBox</tabstop>
  <tabstop>replaceUnderscoresBySpacesCheckBox</tabstop>
  <tabstop>skipCheckBox</tabstop>
  <tabstop>stopCheckBox</tabstop>
  <tabstop>downloadButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
